inc/classes/class_func.php

```diff
@@ -561,21 +621,10 @@ public function Entity2Uml($string)
         return $string;
     }
 
-
-  #### Check Var Content ####
-    public function check_var($var, $type, $min_length, $max_length)
```

Comment:
Fine to remove. 
There is native PHP support with the `is_<type>()` functions, and the error message does not really provide added value, does it? ;)

Reply:
Exactly. Once we are getting more and more stable we can introduce PHP native type system here.